antsibull-docs.cfg

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# Determines whether to add breadcrumbs to plugin docs via hidden sphinx toctrees. This can
+# take up a significant amount of memory if there are a large number of plugins so you can
+# disable this if necessary.
+breadcrumbs = true
+
+# Which docs parsing backend to use. Currently there is only 'ansible-core-2.13', and 'auto'
+# always chooses that one.
+doc_parsing_backend = "auto"
+
+# Determines whether to create the collection index and plugin indexes. They may not be
+# needed if you have a different structure for your website.
+indexes = true
+
+# Determines whether to use HTML blobs for option and return value tables. Using HTML blobs
+# reduces memory and CPU time usage significantly so you can disable this if necessary.
+use_html_blobs = false
+
+# You can specify ways to convert a collection name (<namespace>.<name>) to an URL here.
+# You can replace either of <namespace> or <name> by "*" to match all values in that place,
+# or use "*" for the collection name to match all collections. In the URL, you can use
+# {namespace} and {name} for the two components of the collection name. If you want to use
+# "{" or "}" in the URL, write "{{" or "}}" instead. Basically these are Python format
+# strings (https://docs.python.org/3.8/library/string.html#formatstrings).
+collection_url = {
+  * = "https://galaxy.ansible.com/{namespace}/{name}"
```

Comment:
Note to self: I think we'll need to change this when Galaxy v3 is deployed.

Reply:
Indeed! There should be compatibility redirects for existing collections, so it won't break the docsite for older stable branches, but once new collections are added this can become problematic.

Reply:
I created an issue for this: #147